packages/lesswrong/components/ea-forum/EAGBanner.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+import { createStyles } from '@material-ui/core/styles'
+import { registerComponent, Components } from '../../lib/vulcan-lib';
+import { SECTION_WIDTH } from '../common/SingleColumnSection';
+import { cloudinaryCloudNameSetting } from '../../lib/publicSettings';
+import { Link } from '../../lib/reactRouterWrapper';
+
+const bannerHeight = 250
+const mobileImageId = 'Banner/eagr_banner.png'
+const desktopImageId = 'Banner/eagr_banner.png'
```

Comment:
This image is visibly not high resolution enough, but given the time-sensitivity, I don't think we should push back.